roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Install GPG
+  package:
+    name: gpg
+    state: present
+  tags:
+    - role::filebeat
+
+- name: Install Elasticsearch signing key
+  shell: >-
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch |
+    gpg --yes --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg
+  args:
+    creates: /usr/share/keyrings/elasticsearch-keyring.gpg
+  tags:
+    - role::filebeat
```

Comment:
This, along with the "add elasticsearch repo to apt", should be in a base role used by both roles, to keep deduplication to a minimum.